src/ft2font_wrapper.cpp

```diff
@@ -585,34 +640,36 @@ const char *PyFT2Font_load_char__doc__ =
     "Load character with *charcode* in current fontfile and set glyph.\n"
     "*flags* can be a bitwise-or of the LOAD_XXX constants;\n"
     "the default value is LOAD_FORCE_AUTOHINT.\n"
-    "Return value is a Glyph object, with attributes\n"
-    "  width          # glyph width\n"
-    "  height         # glyph height\n"
-    "  bbox           # the glyph bbox (xmin, ymin, xmax, ymax)\n"
-    "  horiBearingX   # left side bearing in horizontal layouts\n"
-    "  horiBearingY   # top side bearing in horizontal layouts\n"
-    "  horiAdvance    # advance width for horizontal layout\n"
-    "  vertBearingX   # left side bearing in vertical layouts\n"
-    "  vertBearingY   # top side bearing in vertical layouts\n"
-    "  vertAdvance    # advance height for vertical layout\n";
+    "Return value is a Glyph object, with attributes\n\n"
+    "- width          # glyph width\n"
+    "- height         # glyph height\n"
+    "- bbox           # the glyph bbox (xmin, ymin, xmax, ymax)\n"
+    "- horiBearingX   # left side bearing in horizontal layouts\n"
+    "- horiBearingY   # top side bearing in horizontal layouts\n"
+    "- horiAdvance    # advance width for horizontal layout\n"
+    "- vertBearingX   # left side bearing in vertical layouts\n"
+    "- vertBearingY   # top side bearing in vertical layouts\n"
+    "- vertAdvance    # advance height for vertical layout\n";
```

Comment:
Doesn't render with monospace or `pre`, so no need to align.
```suggestion
    "- width: glyph width\n"
    "- height: glyph height\n"
    "- bbox: the glyph bbox (xmin, ymin, xmax, ymax)\n"
    "- horiBearingX: left side bearing in horizontal layouts\n"
    "- horiBearingY: top side bearing in horizontal layouts\n"
    "- horiAdvance: advance width for horizontal layout\n"
    "- vertBearingX: left side bearing in vertical layouts\n"
    "- vertBearingY: top side bearing in vertical layouts\n"
    "- vertAdvance: advance height for vertical layout\n";
```